packages/embed/README.md

```diff
@@ -26,75 +26,73 @@ To use the embed page, you will need the site secret from your Azure Bot Service
 <iframe src="https://webchat.botframework.com/embed/your-bot-id?s=your-site-secret"></iframe>
 ```
 
-**For production use, we strongly prefer you to use token instead of secret.** Please refer to [this article](https://docs.microsoft.com/en-us/azure/bot-service/rest-api/bot-framework-rest-direct-line-3-0-authentication) for how to generate token out of secret.
+**For production use, we strongly recomment that you use token retrieval instead of your secret.** Please refer to [this article](https://docs.microsoft.com/en-us/azure/bot-service/rest-api/bot-framework-rest-direct-line-3-0-authentication) for how to generate token from your secret.
```

Comment:
```suggestion
**For production use, we strongly recommend that you use token retrieval instead of your secret.** Please refer to [this article](https://docs.microsoft.com/en-us/azure/bot-service/rest-api/bot-framework-rest-direct-line-3-0-authentication) for how to generate token from your secret.
```